packages/cli/src/commands/dev.ts

```diff
@@ -4,19 +4,33 @@ import { showWelcomeScreen } from './init.consts';
 import * as ora from 'ora';
 import * as open from 'open';
 import * as chalk from 'chalk';
+import { SERVER_HOST } from '../constants';
 
 process.on('SIGINT', function () {
-  console.log('Caught interrupt signal');
   // TODO: Close the NTFR Tunnel
   process.exit();
 });
 
+export enum CloudRegionEnum {
+  US = 'us',
+  EU = 'eu',
+  STAGING = 'staging',
+}
+
+export enum DashboardUrlEnum {
+  US = 'https://dashboard.novu.co',
+  EU = 'https://eu.dashboard.novu.co',
+  STAGING = 'https://dev.dashboard.novu.co',
+}
+
+const TUNNEL_URL = 'https://novu.sh/api/tunnels';
```

Comment:
I won't release the new package until this new `novu.sh` tunnel service is available.